### PR TITLE
travis: add go1.10

### PR DESCRIPTION
**Pull request by ferhatelmas**
might want to drop 1.7 too but didn't before getting confirm.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,4 +6,5 @@ go:
   - 1.7
   - 1.8
   - 1.9
+  - 1.10
```
Comment (thoas) [inline .travis.yml]: add `"` around, it's a `[known issue](https://github.com/travis-ci/travis-ci/issues/9247)

Comment (ferhatelmas) [reply .travis.yml]: TIL, updated.

Comment (ferhatelmas) [reply .travis.yml]: also, added `tip`.

**Comment by thoas** [comment]
thank you @ferhatelmas :)

**Pull request merged**